dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -118,6 +119,10 @@
     @ApiModelProperty(value = "This indicates the DOI status", position = 11)
     private DOIStatus doiStatus;
 
+    @ApiModelProperty(value = "Particularly for hosted workflows, this records who edited to create a revision", position = 12)
```

Comment:
WorkflowVersion and Tag also have a position 12, what happens?

Reply:
Based on the swagger, looks like they both get 12 :confused: